tests/compile-test.rs

```diff
@@ -217,7 +217,7 @@ fn run_ui(cx: &TestContext) {
         vec![config],
         ui_test::default_file_filter,
         ui_test::default_per_file_config,
-        status_emitter::Text::from(cx.args.format),
+        Box::<dyn ui_test::status_emitter::StatusEmitter>::from(cx.args.format),
```

Comment:
nit: could you import `ui_test::status_emitter::StatusEmitter`? `Box::<dyn StatusEmitter>::from(…)` looks just a bit less ugly, especially when it is repeated 4 times.

Reply:
done :)